modules/ROOT/pages/field-level-capabilities.adoc

```diff
@@ -0,0 +1,256 @@
+= Field Level Capabilities
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+There are several features for Java SDK that support field level capabilities
+for value providers and sample data. Using these features enables you to define
+value providers for parameter fields, or to use parameter fields as acting
+parameters for either a value provider or sample data.
+
```

Comment:
I wonder if we should include here some narrative that explicitly mentions each of the subtopics included in this topic set, to prepare the audience for what is to come and help them to decide if this is the topic set that they need to read?